sdk/src/asset_handlers/sfnt_io.rs

```diff
@@ -891,15 +903,15 @@ where
     TWriter: Read + Seek + ?Sized + Write,
 {
     // Read the font from the input stream
-    let mut font = SfntFont::from_reader(input_stream).map_err(|_| FontError::LoadError)?;
+    let mut font =
+        SfntFont::from_reader(input_stream).map_err(|_| FontError::DeserializationError)?;
```

Comment:
Is this mapping necessary?  They both throw the same error type right?

Reply:
Yeah, I was curious about this too. The previous code was basically remapping the underlying error types into a `DeserializationError`. I wonder if we do the same thing here as the `FontSaveError`, have a `FontLoadError` with the other details in it to capture that the fact it failed to load with more details why it failed to load. What do you think?

Reply:
Uh.. I don't know if it's necessary; the `from_reader()` should give you all the context of why it's failing - adding more information here would just give.. context about why you were reading? Might not be all that helpful

Reply:
Ok, thanks. This will just remove the `DeserializeError` then, updating...